.github/workflows/docker_test.yaml

```
@@ -0,0 +1,18 @@
+name: Docker build test
+on: pull_request
+
+jobs:
+  test_docker_image_build:
+    name: Test building Docker image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        id: checkout_code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)